docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-2.md

```diff
@@ -150,12 +150,21 @@ metadata:
 spec:
   locationSelectors:
   - matchLabels: {"env":"prod"}
-  namespaceSelector:
-    matchLabels: {"common":"si"}
-  nonNamespacedObjects:
+  downsync:
+  - apiGroup: ""
+    resources: [ configmaps ]
+    namespaceSelectors:
+    - matchLabels: {"common":"si"}
+    objectNames: [ httpd-htdocs ]
+  - apiGroup: apps
+    resources: [ replicasets ]
+    namespaceSelectors:
+    - matchLabels: {"common":"si"}
+    objectNames: [ "*" ]
   - apiGroup: apis.kcp.io
```

Comment:
If I want to downsync all objects in a namespace (as it was previously) can I just provide something like:
```
downsync
 - namespaceSelectors:
    -  matchLabels: {"common":"si"}
```
or do I need to provide something like:
```
 downsync:
  - apiGroup: "*"
    resources: [ "*" ]
    namespaceSelectors:
    - matchLabels: {"common":"si"}
    objectNames: ["*"]
```

Reply:
This is an interface _change_.  The old way of specifying namespaced objects to downsync is no longer available.  The comment on the new type, DownsyncObjectTest, gives its precise semantics.